test/controllers/articles_controller_test.rb

```diff
@@ -3,85 +3,143 @@
 require "test_helper"
 
 # Controller tests for news articles
-class ArticleControllerTest < FunctionalTestCase
-  def test_create_article_get
+class ArticlesControllerTest < FunctionalTestCase
+  ############ test Actions that Display data (index, show, etc.)
+
+  def test_index
```

Comment:
I would split this test up into three separate tests - one for each `get`.

Reply:
I don't see `test_index` getting broken up.  Was this intentional?

Reply:
Thanks for taking the time to review this again.
I think GitHub's incomplete file display obscures some changes.
I did break the former `test_index` into 3 tests, 1 get/test:
`test_index`, `test_index_filtered`, `test_index_links_to_creates`
It might be easiest to see here: https://github.com/MushroomObserver/mushroom-observer/pull/629/commits/0fe65e28ebb4c4d3ccfc6c8f22671576803281b1

Reply:
Ah, I see it now.  Sorry about that.